<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNewLib</class>
 <widget class="QDialog" name="DialogNewLib">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新建</string>
  </property>
  <widget class="QRadioButton" name="RbSingleTerm">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>单端</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="RbMultiTerm">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>多端</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>功能定义：</string>
   </property>
  </widget>
  <widget class="QLabel" name="LbFuncDefine">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>47</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnFuncSet">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>46</y>
     <width>23</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="LbFileName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>83</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>库文件名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="LbPattern">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>117</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>样式：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CbPattern">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>115</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="BtnOK">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确 定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnCancel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取 消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EbLibFileName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
